.github/workflows/crowdin-upload.client-ui.yml

```
@@ -2,8 +2,8 @@ name: i18n - Upload Client UI
 on:
   workflow_dispatch:
   schedule:
-    # runs every weekday at 11:15 AM UTC
-    - cron: '15 11 * * 1-5'
+    # runs every weekday at 7:00 AM UTC
+    - cron: '0 7 * * 1-5'
```

```suggestion
    # runs every weekday at 7:15 AM UTC
    - cron: '15 7 * * 1-5'
```

Cron jobs fired on the hour are more likely to fail.